src/universal/containers/Action/ActionContainer.js

```diff
@@ -4,10 +4,7 @@ import React, {Component} from 'react'
 import {withRouter} from 'react-router-dom'
 import Action from 'universal/components/Action/Action'
 import globalStyles from 'universal/styles/theme/globalStyles'
-import * as E from 'emotion-server'
-import A from 'universal/Atmosphere'
 
-@withRouter
```

Comment:
I gotta be honest, I like decorators. I wish the standard would have gone this way

Reply:
it still might...but i doubt it. the champion is lumping decorators in with private methods & some other stuff. it'll be a hard to get passed: https://github.com/tc39/proposal-decorators